pytket/phir/place_and_route.py

```diff
@@ -16,6 +17,8 @@
     from .machine import Machine
     from .sharding.shard import Cost, Ordering, Shard, ShardLayer
 
+logger = logging.getLogger(__name__)
```

Comment:
Do you need these logger lines in all these files for this change?

Reply:
I'd suggest adding some `logger.info()`s and `logger.debug()`s in this file if they have helped you during this PR as they may help others in future. Otherwise, no need to include them.

Reply:
ok, it was pretty specific stuff that would probably just clutter a log file and make it harder to read, ill just take them out